### ISSUE TITLE
Policy/request.go has faulty DelayAction struct.

### ISSUE THREAD

**Issue by jaceq** [opsgenie/opsgenie-go-sdk-v2#29]
Here: https://github.com/opsgenie/opsgenie-go-sdk-v2/blob/d4ed1a6ec1b3140530679f1b394b50c6f1b3dc31/policy/request.go#L550
is a bug, following struct is configured:
```
type DelayAction struct {
        DelayOption DelayType `json:"delayOption,omitempty"`
        UntilMinute int       `json:"untilMinute,omitempty"`
        UntilHour   int       `json:"untilHour,omitempty"`
        Duration    *Duration `json:"duration,omitempty"`
}
```

Problem is that if UntilMinute and / or UntilHour is 0 then argument is omitted as it's seen as empty (which in this case is not the case). 
Due to this issue it's not possible to set any of those to 0.
According to: https://www.sohamkamani.com/blog/golang/2018-07-19-golang-omitempty/

Solution would be to switch field to:
```
 *int 
```

So, struct would look like that:
```
type DelayAction struct {
        DelayOption DelayType `json:"delayOption,omitempty"`
        UntilMinute *int       `json:"untilMinute,omitempty"`
        UntilHour   *int       `json:"untilHour,omitempty"`
        Duration    *Duration `json:"duration,omitempty"`
}
```

**Comment by freeseacher** [comment]
any update on this ?

**Comment by jaceq** [comment]
It seems someone  did a PR which would solve this: https://github.com/opsgenie/opsgenie-go-sdk-v2/pull/40

**Comment by MeralBusraTekinsen** [comment]
We have released v1.1.1 for this fix. Thanks.
